test/distributed/test_inductor_collectives.py

```diff
@@ -905,6 +911,100 @@ def func(inp, *, tag, ranks, group_size):
         assert same(out, correct), f"{out} va {correct}"
 
 
+class TestDTensorCompile(torch._dynamo.test_case.TestCase):
+    def setUp(self):
+        super().setUp()
+        fake_store = FakeStore()
+        dist.init_process_group(
+            "fake", store=fake_store, rank=0, world_size=self.world_size
+        )
+
+    def tearDown(self):
+        super().tearDown()
+        dist.destroy_process_group()
+
+    @property
+    def device_type(self) -> str:
+        return "cuda" if torch.cuda.is_available() else "cpu"
+
+    @property
+    def world_size(self) -> int:
+        return 4
+
+    @patch.object(torch._inductor.config, "reorder_for_compute_comm_overlap", True)
+    def test_2d_fsdp_tp_compile(self):
```

Comment:
nit: can we put this as a test case inside `test_dtensor_compile.py` instead of here as it touches higher level DTensor APIs instead of "collectives"?